src/main/java/Pages/PlaylistPage.java

```diff
@@ -0,0 +1,87 @@
+package pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import java.util.List;
+
+    public class PlaylistsPage extends BasePage {
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/230855736-c4965d69-04eb-4a13-802f-ebd972b2b457.png)

![image](https://user-images.githubusercontent.com/104761482/230855823-129c0653-8531-4e88-bfa6-379ad01f39e4.png)

![image](https://user-images.githubusercontent.com/104761482/230855863-c61711f7-c032-4cbf-b882-60baa812fa05.png)

![image](https://user-images.githubusercontent.com/104761482/230855904-b68476aa-b6c0-416c-a3bc-d5c8aca4a576.png)

![image](https://user-images.githubusercontent.com/104761482/230856000-f93d7334-dc48-4bbf-bd97-fb62b9fec968.png)

![image](https://user-images.githubusercontent.com/104761482/230856126-738f8c57-9c4e-4c04-9dad-e2ac6bf0c6aa.png)